src/components/gridHOC.jsx

```diff
@@ -25,6 +26,10 @@ type Props = {
   pageSize: number,
   cellComponent: Component<*>,
   editButtonLabel: string,
+  exportable: boolean,
+  exportData: Array<Object>,
```

Comment:
What is exportData? How is it different from the data in the grid (which comes from the redux-form injected `data` prop, and is stored in component state here). If each value needs some formatting, you should be able to pass in a `formatter` with each column.